apps/mini-rx-angular-demo/src/app/modules/pixel-art/state/art-store.service.ts

```diff
@@ -20,13 +20,13 @@ export class ArtStoreService extends ComponentStore<ArtState> {
 
         // This Observable is passed to setState: therefore the Observable has to emit ArtState
         // FYI: the typing of setState would also allow Observable<Partial<ArtState>>
-        const delayedOpacity$: Observable<ArtState> = timer(Math.random() * 5000).pipe(
-            map(() => ({ opacity: Math.random() }))
+        const delayedOpacity$: Observable<number> = timer(Math.random() * 5000).pipe(
+            map(() => Math.random())
         );
 
         // You could use JS setTimeout, but that approach would require some cleanup code to cancel the timer when the component destroys
         // setState with Observable manages cleanup (of subscriptions) internally
```

Comment:
This comment should probably be removed or reworked since you're no longer calling `setState`?

Reply:
comment reworked